### ISSUE TITLE
Cognizance 5 should be skipped over if in Cognizance 4 you destroyed the Korath fleet

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#5944]
**Is your feature request related to a problem? Please describe.**
The player has the choice to destroy the Korath fleet in Cognizance 4, and the story will acknowledge this action. But Cognizance 5 contains the exact same Korath fleet (not just in composition but in story as well), which doesn't make much sense if you had destroyed it.

**Related Issue Links**
#1263 

**Describe the solution you'd like**
Have Cognizance 5 be skipped over if you destroyed the fleet in Cognizance 4.

**Describe alternatives you've considered**
Say in the story that the Cognizance 5 fleet is separate from the Cognizance 4 fleet.

**Additional context**
N/A

**Comment by Amazinite** [comment]
So Chilia does say that there may be more fleets like that one if you destroy the fleet in Cognizance 4, yet Cognizance 5 still acts as though the fleet you fight is the same one from Cognizance 4, evident by the fact that you're first sent to the system that you found the fleet in during Cognizance 4 and get a dialog box saying they left.

**Comment by Ornok** [comment]
It does look like the player is killing the same fleet twice. If we skip Cognizance 5, we also need to skip/adapt the first part of Cognizance 6.


Another possibility to avoid changing Cognizance 6:

Instead of completely skipping Cognizance 5, we could have the "fleetkilled" branch task the player with patrolling the systems between Aescolanus and Nenia with a Remnant escort to make sure no _other_ Korath fleet is present. Sure enough they will find one and have to destroy it.
If we want to keep the current  `on enter` dialog, we just branch into a parallel mission (Cognizance 5A?). That would also allow the new Korath fleet to actually be different from the first one.

**Comment by Zitchas** [comment]
The intent was to have modified dialog along the lines of "Great initiative taking down that fleet, but it wasn't the whole thing. Go kill the rest." and otherwise have the missions play through as normal.

**Comment by Ornok** [comment]
Yes, that shows in the conversation with Chilia if you destroyed the fleet in Cognizance 4.

To be honest I don't think most player would be bothered by (or even notice) the second fleet being the same one (in composition) as the one in Cognizance 4. 
The only thing that actually points to it being the same one (story-wise) is the "on enter" dialog:
`The Korath fleet is gone. Given the urgency Prefect Chilia put on this mission, the onus is on you to find it... Quickly.`

We could change it to something that fits both branches, maybe like:
`Your sensors don't pick up any Korath ship in this system. Given the urgency Prefect Chilia put on this mission, the onus is on you to find them... Quickly.`

**Comment by Amazinite** [comment]
Taking out a sizeable Korath fleet isn't the type of optional action that should just result in a simple dialog change, I'd think.
